core/domain/fs_domain.py

```diff
@@ -147,7 +147,7 @@ def get(self, filepath):
 
         Returns:
             FileStream or None. It returns FileStream
-                domain object if the file exists. Otherwise, it returns None.
+            domain object if the file exists. Otherwise, it returns None.
```

Comment:
Can probably move some of this to the previous line.

Reply:
Done